client/settings/general-settings/style.scss

```diff
@@ -0,0 +1,42 @@
+.general-settings {
+	&__bank-information {
+		display: flex;
+		flex-direction: column;
+
+		> :first-child {
+			flex-grow: 1;
+		}
+
+		@include breakpoint( '>800px' ) {
+			align-items: center;
+			justify-content: space-between;
+			flex-direction: row;
+			flex-wrap: nowrap;
+		}
+	}
+
+	&__account-statement-wrapper {
+		position: relative;
+
+		input[maxlength='22'] {
+			// to make room for the help text, so that the input's text and the help text don't overlap
+			padding-right: 55px;
```

Comment:
Looks good but the input was pushed to the left as well.

Reply:
Ungh, sorry for the confusion! Looks like what I did was testing if I can replace the `input[maxlength='22']` selector with `.general-settings__account-statement-input` and leaving this new selector in place.

The `.general-settings__account-statement-input` selector targets the input's container which caused everything in the container--including the input--to be pushed to the left (rather than just the input's contents).

This means that there is no bug after all. :) Sorry again!